qutebrowser/misc/utilcmds.py

```diff
@@ -369,8 +369,11 @@ def nop():
 
 @cmdutils.register()
 @cmdutils.argument('win_id', win_id=True)
-def version(win_id):
+def version(win_id, paste=False):
```

Comment:
You'll need to add docs for the argument, see e.g. the `debug_log_filter` docstring above.